int64 type
---
bool success
string message